dev/io.openliberty.netty.internal.impl/test/io/openliberty/netty/internal/impl/NettyFrameworkImplTest.java

```diff
@@ -169,6 +170,91 @@ protected void channelRead0(ChannelHandlerContext ctx, ByteBuf buf) throws Excep
         Assert.assertTrue(framework.getActiveChannels().contains(testChannels.get(0)));
     }
 
+    /**
+     * Create an outbound UDP channel
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void testUDPStartOutbound() throws Exception {
+        BootstrapExtended bootstrap = framework.createUDPBootstrapOutbound(null);
+        bootstrap.handler(new SimpleChannelInboundHandler<ByteBuf>() {
+            @Override
+            protected void channelRead0(ChannelHandlerContext ctx, ByteBuf buf) throws Exception {
+                // do nothing
+            }
+        });
+        final CountDownLatch latch = new CountDownLatch(1);
+        framework.startOutbound(bootstrap, "*", 0, future -> {
+            if (future.isSuccess()) {
+                testChannels.add(future.channel());
+                latch.countDown();
+            } else {
+                Assert.fail("framework failed to start: " + future.cause().getMessage());
+                latch.countDown();
+            }
+        });
+        framework.setServerStarted(null);
+        latch.await(PORT_BIND_TIMEOUT_MS, TimeUnit.MILLISECONDS);
+        Thread.sleep(10); // wait to ensure framework's internal callbacks are handled
```

Comment:
If this is run as the unit tests in a build you might want to add a good slug into 10ms to cope with fyre sluggishness?

Reply:
Nice idea, updated to 1000ms